bin/hdfcoinc/pycbc_page_foundmissed

```diff
@@ -201,6 +210,7 @@ ax = plot.gca()
 plot.xlabel(labels[args.axis_type])
 plot.ylabel(labels[args.distance_type])
 plot.grid()
+plot.tight_layout()
```

Comment:
I am debated on the `tight_layout()` here. It will definitely tidy up a *particular* plot, but it will also make the plot change slightly every time you make it with different input data. This slightly spoils the "flipping between two plots" feature that many of us love. Up to you though, especially if it is a mess to get a tidy plot without `tight_layout()`.

Reply:
Good point, I'll remove this call - I hadn't realised it would ruin the flipping-between plots thing, which is probably one of the things this is used for most